File: templates/Uwp/_comp/MVVMLight/Feat.IdentityOptionalLogin/ViewModels/SettingsViewModel_postaction.cs
//{[{
using GalaSoft.MvvmLight;
using GalaSoft.MvvmLight.Command;
//}]}

namespace Param_RootNamespace.ViewModels
{
    public class SettingsViewModel : ViewModelBase
    {
        public bool IsBusy
        {
            get => _isBusy;
            set
            {
                Set(ref _isBusy, value);
//^^
//{[{
                LogInCommand.RaiseCanExecuteChanged();
                LogOutCommand.RaiseCanExecuteChanged();
//}]}
            }
        }
    }
}

File: templates/Uwp/_comp/MVVMLight/Service.IdentityOptionalLogin._VB/ViewModels/SettingsViewModel_postaction.vb
'{[{
Imports GalaSoft.MvvmLight
Imports GalaSoft.MvvmLight.Command
'}]}
Namespace ViewModels
    Public Class SettingsViewModel
        Inherits ViewModelBase

        Public Property IsBusy As Boolean
            Get
                Return _isBusy
            End Get
            Set(value As Boolean)
                [Set](_isBusy, value)
'^^
'{[{
                LogInCommand.RaiseCanExecuteChanged()
                LogOutCommand.RaiseCanExecuteChanged()
'}]}
            End Set
        End Property
    End Class
End Namespace
